.github/workflows/ci.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         include:
           - ruby: '3.0'
             os: ubuntu-latest
-            rails: rails_71
+            rails: rails_70
```

Comment:
If this solution to https://github.com/activeadmin/activeadmin/pull/8488#issuecomment-2379322266 is acceptable, I will change the required actions in settings to allow the branch to be merged without skipping branch rules

Reply:
I'm fine with dropping Ruby 3.0 from CI now. If that helps here and you are ok with it, let's do that.

Reply:
I would not do that if 3.0 is supported, I'm afraid of unintended breaking changes at patch level, in example by using 3.1 only syntax (like `Array#intersect?`)

This happened several times in the past, a couple of the most recent breaking changes at patch level that I remember are

- sidekiq-scheduler/sidekiq-scheduler#477
- thoughtbot/factory_bot#1614

Please let me know what you think

Reply:
Well I figure we would bump the minimum required version anyway. I recall we've done that each time we've dropped an old EOL version. You are welcome to open that PR to drop Ruby 3.0 now if you want.

Reply:
As a reference, here it is another breaking change introduced in a patch level release because of the shorthand syntax (>= 3.1)

rails/importmap-rails#268